pkgs/jnigen/test/simple_package_test/runtime_test_registrant.dart

```diff
@@ -702,6 +702,44 @@ void registerTests(String groupName, TestRunnerCallback test) {
           expect(fortyTwo.intValue(releaseOriginal: true), 42);
         });
       });
+      for (final style in ['callback', 'implemented class']) {
+        test('Listener callbacks - $style style', () async {
+          final completer = Completer<void>();
+
+          final MyRunnable runnable;
+          if (style == 'callback') {
+            runnable = MyRunnable.implement($MyRunnable(
+              run: completer.complete,
+              run$async: true,
```

Comment:
`blocking: false`

Maybe only generate `$blocking` as a variable name if `blocking` is a member already taken.

Reply:
It's per method, so it's going to be `run$blocking`. The quip with blocking is that the default is `true`, I prefer boolean defaults to be `false` instead.